Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -47,8 +51,9 @@ import com.materialstudies.reply.util.setOutgoingTransitions
 import kotlin.LazyThreadSafetyMode.NONE
 
 class MainActivity : AppCompatActivity(),
-    Toolbar.OnMenuItemClickListener,
-    NavController.OnDestinationChangedListener {
+                     Toolbar.OnMenuItemClickListener,
+                     NavController.OnDestinationChangedListener,
+                     NavigationAdapter.NavigationAdapterListener {
```

Comment:
Oh interesting. 

If I remember correctly, the initial though was to be generic in case we add large-screen layout support with Navigation Rail. Navigation Rail respect the same interface interface and MainActivity wouldn't have to change much/any logic.

We could rename it for the time being and then change the name to something more adaptable if we update the sample in the future. 

Thoughts?

Reply:
ah ok, was wondering if that was the reason but then i didn't see the bottom nav/nav rail abstraction in the code yet. i think it's fine as is if that's the plan then.